coalib/results/Diff.py

```diff
@@ -375,12 +375,25 @@ def add_lines(self, line_nr_before, lines):
 
         linediff = self._get_change(line_nr_before, min_line=0)
         if linediff.add_after is not False:
-            raise ConflictError('Cannot add lines after the given line since '
-                                'there are already lines.')
+            raise ConflictError('Cannot add line(s) after the given line since '
+                                'there are already line(s).')
```

Comment:
what I mean is that the sentence, though it's written with the plural form of `line` -> `lines`, `lines` is meant generically. It's like a general condtition that holds:
"You can't generally add lines (whether these are 1 or many), when there are already lines (1 or many)"

I know this is slightly nitpicking, though I'm not a fan of making changes that aren't necessary for the commit^^ And in this case I believe the grammar is totally okay, whether a single or multiple lines are meant.